docs/dev/plugins.rst

```diff
@@ -9,6 +9,7 @@ Plugins
    - :ref:`plugins generic`
 
 Plugins can extend or replace functionality of various components of searx.
+To create a plugin put your python file in searx > plugins.
```

Comment:
No need to modify to plugins.rst since it is now a core feature.

Reply:
Oh yeah good point! Totally forgot that I documented that. Will remove

Reply:
Fixed it!